Sources/PCRoom.swift

```diff
@@ -8,6 +8,7 @@ public final class PCRoom {
     public let createdByUserID: String
     public let createdAt: String
     public internal(set) var updatedAt: String
+    public internal(set) var deletedAt: String?
```

Comment:
We also will want to add an accompanying `public var deletedAtDate: Date { return self.dateFormatter.date(from: self.deletedAt)! }` (to match `createdAt` and `updatedAt`)